pyproject.toml

```diff
@@ -1,3 +1,51 @@
+[build-system]
+requires = ["setuptools>=61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+version = "v2.1.1"
+name = "PyGithub"
+authors = [{ name = "Vincent Jacques", email = "vincent@vincent-jacques.net" }]
+description = "Use the full Github API v3"
+readme = "README.md"
+requires-python = ">=3.7"
+keywords = ["github"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Web Environment",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: GNU Library or Lesser General Public License (LGPL)",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Software Development",
+]
+dependencies = [
```

Comment:
The `setup.py` had `extras_require` and `tests_require`. Is there something similar in `pyproject.toml`? Do current commands to install pygithub continue to work with this change? Or is this a breaking change for users (and CIs)?

Reply:
`extras_require` is `project.optional-dependencies` which won't break user.

I don't think there is a `tests_require` with pep517.